docs/topics/forms/formsets.txt

```diff
@@ -264,6 +259,15 @@ itself. When rendering a formset in a template, you can include all
 the management data by rendering ``{{ my_formset.management_form }}``
 (substituting the name of your formset as appropriate).
 
+.. note::
+
+    As well as the ``form-TOTAL_FORMS`` and ``form-INITIAL_FORMS`` fields shown
+    in the examples here, the management form also includes
+    ``form-MIN_NUM_FORMS`` and ``form-MAX_NUM_FORMS`` fields. The are output
+    with the rest of the management form, but only for the convenience of
+    client-side code. These fields are not required, and so are not shown in
```

Comment:
Grammar nit: I don't think this comma is necessary. Either drop the comma (my preference) or keep the comma and add a subject to the 2nd clause (e.g. "..., and so they are ...").